tools/bridge/bridge/confirmation_task_planner.py

```diff
@@ -29,43 +28,46 @@ def __init__(
         self.confirmation_task_queue = confirmation_task_queue
 
     def run(self):
-        self.logger.info("Starting")
+        logger.debug("Starting")
         try:
             greenlets = [
                 gevent.spawn(self.process_transfer_events),
                 gevent.spawn(self.process_home_bridge_events),
             ]
             gevent.joinall(greenlets, raise_error=True)
         finally:
-            self.logger.info("Stopping")
+            logger.info("Stopping")
```

Comment:
```suggestion
            logger.debug("Stopping")
```

to use the same level as `Starting`